pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -545,7 +546,29 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 			return conf
 		},
 		errMsg: "invalid configuration: imageMaximumGCAge 1ns must be greater than imageMinimumGCAge 2ns",
-	}}
+	}, {
+		name: "pod logs path must be absolute",
```

Comment:
One question. What happens if the entire path does not exist? Or we don't have permission to create a directory in that path?

Not sure if validation is the place to check but wanted to ask about that.

Reply:
NVM, I see where we create this folder in kubelet.go.